src/test/scala/symsim/concrete/ConcreteSarsaIsSarsaSpec.scala

```diff
@@ -6,6 +6,9 @@ import symsim.examples.concrete.mountaincar.MountainCar
 class ConcreteSarsaIsSarsaSpec
   extends SymSimSpec:
 
+  implicit override val generatorDrivenConfig =
```

Comment:
change `implicit` to `given`

Reply:
And we should move this to `SymSimSpec`

Reply:
I moved it into SymSimSpec and it was fine but when I changed given to implicit, it started to complaining.
I am trying to solve the issue but I thought maybe there is some different syntax that you already know.